docs/basics/part-8a-using-earthly-in-your-current-ci.md

```diff
@@ -0,0 +1,93 @@
+Earthly was designed to be used as the main CI build specification. It can be integrated with any CI system and used instead of the CI's own native syntax.
+
+The main benefit of using Earthly in an existing CI is that it provides consistency between local and CI builds. This allows you to debug, test, and iterate on your CI builds locally, before pushing them to the CI. The fact that Earthly uses containers underneath provides a level of guarantee that the build behaves the same no matter where it is run from. This is what we call **repeatability**.
+
+In this section, we will explore how to use Earthly in a well-known CI system, such as GitHub Actions. For more information on how to use Earthly in other CIs such as GitLab, Jenkins, or CircleCI, you can check out the [CI Integration page](../ci-integration/overview.md).
+
+## Using Earthly in your current CI
+
+To use Earthly in a CI, you typically encode the following steps in your CI's build configuration:
+
+1. Download and install Earthly
+2. Log in to image registries, such as DockerHub
+3. Run Earthly
+
+For example, here is how this would work in GitHub Actions:
+
+```yaml
+# .github/workflows/ci.yml
+
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+      DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+      FORCE_COLOR: 1
+    steps:
+    - uses: earthly/actions/setup-earthly@v1
+      with:
+        version: v0.7.0
+    - uses: actions/checkout@v2
+    - name: Put back the git branch into git (Earthly uses it for tagging)
+      run: |
+        branch=""
+        if [ -n "$GITHUB_HEAD_REF" ]; then
+          branch="$GITHUB_HEAD_REF"
+        else
+          branch="${GITHUB_REF##*/}"
+        fi
+        git checkout -b "$branch" || true
+    - name: Docker Login
+      run: docker login --username "$DOCKERHUB_USERNAME" --password "$DOCKERHUB_TOKEN"
+    - name: Run build
+      run: earthly --push +build
```

Comment:
Should this be `earthly --ci --push +build` as referenced below?

Reply:
Good catch!